pkg/config/config.go

```diff
@@ -893,6 +893,13 @@ func InitConfig(config Config) {
 	// Time in seconds
 	config.BindEnvAndSetDefault("logs_config.file_scan_period", 10.0)
 
+	// Controls how wildcard file log source are prioritized when there are more files
+	// that match wildcard log configurations than the `logs_config.open_files_limit`
+	// Choices are 'prioritizeNewest', 'simpleAlpha'
+	// WARNING: 'prioritizeNewest' is less performant than 'simpleAlpha' and will trigger
+	// more disk I/O at the wildcard log paths
```

Comment:
Would it make sense to document what these options do in more detail?

Nit pick, but I'm not a fan of these names. What if we borrow from a file manager lingo and call them `sortByName`, `sortByModifiedTime`?

Reply:
Yes these should definitely be documented a bit more, is this the best place to do that?

I'm open to those new names, I was kind of avoiding `sortBy*` names because this changes the behavior slightly more than just sorting (greedy vs global log source consumption).

I guess the trade-off is a simpler name that isn't 100% accurate vs weird names that require you to check the docs.
I have no _strong_ preference here, curious what your thoughts are.

Reply:
We can document them right here in the template.

I feel that global vs greedy is a relatively minor detail that doesn't necessarily need to have impact on the option name, provided that it is documented. Maybe it would sound better if we drop "sort" from the name, and call them `byName`, `byModificationTime`?

And perhaps in snake case too, for consistent style with few other enumerations already in the config.